internal/core/command/init.go

```diff
@@ -31,9 +31,6 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
 )
 
-// Global variables
-var Configuration = &ConfigurationStruct{}
-
 // Global ErrorConcept variables
 var httpErrorHandler errorconcept.ErrorHandler
```

Comment:
https://github.com/edgexfoundry/edgex-go/issues/2016#issuecomment-550421660

Agreed previously to defer this to #2112.  Reconsidered and want you to remove this global as part of this PR (since this is the last PR to remove service-specific globals).  #2112 won't "redo" this work as it'll have to implement a non-global httpErrorHandler.

Will have a minor knock on effect since we won't need a call to `errorconcept.NewErrorHandler` and that's the only usage of `loggingClient`.